### PR TITLE
Changed package names in order to publish

### PR DESCRIPTION
**Pull request by edewit**
Added documentation version and docs to be able to publish these packages

- **changed name and added version number**
- **added readme for package usage**

### CODE REVIEW

**Review by ssilvert — CHANGES_REQUESTED** [review]
I'm wondering about how we deal with the names we have now exported.  

I think we at least should guarantee that code derived from our account ui will not break from a minor release.

Any ideas on how we make sure not to change what is essentially now a public API?  Maybe we could create tests that use all the names so code won't build if we change an exported name?

**Comment by edewit** [comment]
> I'm wondering about how we deal with the names we have now exported.
> 
It's just a component name, even if we would want to change that at some point internally we could still export it under a different alias

> I think we at least should guarantee that code derived from our account ui will not break from a minor release.
> 
> Any ideas on how we make sure not to change what is essentially now a public API? Maybe we could create tests that use all the names so code won't build if we change an exported name?

I don't see how this is different from what we already have with the other customisation features. I would suggest to make a quickstart that get tested each with each release

**Review by jonkoops — COMMENTED** [review]
Let's hold off on this until we move to PatternFly 5, as I see a lot of merge conflicts waiting to happen.

File: js/apps/account-ui/package.json
Comment (jonkoops) [inline js/apps/account-ui/package.json:3]: If this package is published we should use `999.0.0-SNAPSHOT` as a version, that will then be replaced by the actual release version when the release is ran.
```suggestion
  "version": "999.0.0-SNAPSHOT",
```

**Review by jonkoops — CHANGES_REQUESTED** [review]
Got one small thing, but otherwise LGTM, there are also some small linting errors that will need to be fixed for CI to pass.

**Review by jonkoops — APPROVED** [review]

**Review by ssilvert — APPROVED** [review]

**Pull request merged**
